migrations/scripts/lts-to-lts/0.23-to-0.25/playbooks/tasks/rollback.yaml

```diff
@@ -52,11 +52,11 @@
     regex: 'postgres(ql)?:\/\/(?P<username>[^:]+):(?P<password>[^@]+)@(?P<hostname>[^:]+):(?P<port>\d+)'
 
 - debug:
-    msg: "DB username: {{ db_username }}"
-- debug:
-    msg: "DB hostname: {{ db_hostname }}"
-- debug:
-    msg: "DB port: {{ db_port }}"
+    msg:
+      - "DB hostname: {{ db_hostname }}"
+      - "DB port: {{ db_port }}"
+      - "DB username: {{ db_username }}"
+      - "DB password (sha512): {{ db_password | password_hash('sha512') }}"
```

Comment:
do you want to log the password like this? What is the need to do that?

Reply:
good catch. i had put this in while troubleshooting with the intent to revert before pushing. i'll go ahead and remove that, especially given that this is not part of the issue i was troubleshooting.